plan/logical_plans.go

```diff
@@ -388,7 +395,7 @@ func (ds *DataSource) deriveIndexPathStats(path *accessPath) error {
 		}
 		path.countAfterIndex = math.Max(path.countAfterAccess*selectivity, ds.statsAfterSelect.count)
 	}
-	return nil
+	return path.index.Unique && path.eqCondCount == len(path.index.Columns), nil
```

Comment:
It's better to only check where the ranges are all point ranges, just like `deriveTablePathStats`. We can check `path.index.Unique` in `deriveStats()`